pkg/webui/components/footer/index.js

```diff
@@ -31,6 +33,10 @@ const m = defineMessages({
   footer: "You are the network. Let's build this thing together.",
 })
 
+const isMinimized =
+  Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0) <=
```

Comment:
`window.innerWidth >= document.documentElement.clientWidth` will always be `true`.
`innerWidth` and `clientWidth` are equal when there is no scrollbar, or `innerWidth > clientWidth` because `clientWidth` accounts for scrollbar and computes only the visible part of the screen (document)